mantidimaging/gui/windows/operations/test/test_presenter.py

```diff
@@ -361,3 +365,23 @@ def test_running_operation_records_previous_button_states(self, _):
         self.presenter._do_apply_filter(None)
         assert self.presenter.prev_apply_single_state == prev_apply_single_state
         assert self.presenter.prev_apply_all_state == prev_apply_all_state
+
+    def test_init_crop_coords_does_nothing_when_stack_is_none(self):
+        mock_roi_field = mock.Mock()
+        self.presenter.init_crop_coords(mock_roi_field)
+        mock_roi_field.setText.assert_not_called()
+
+    def test_init_crop_coords_does_nothing_when_image_is_greater_than_200_by_200(self):
+        mock_roi_field = mock.Mock()
+        self.presenter.stack = mock.Mock()
+        self.presenter.stack.presenter.images.data = np.ones((2, 201, 201))
+        self.presenter.init_crop_coords(mock_roi_field)
+        mock_roi_field.setText.assert_not_called()
+
+    @parameterized.expand([(190, 201, "0, 0, 200, 95"), (201, 80, "0, 0, 40, 200"), (200, 200, "0, 0, 100, 100")])
```

Comment:
Could this be done with pytest to save needing a new dependency? https://docs.pytest.org/en/stable/parametrize.html

Reply:
This doesn't seem to work when the class inherits from `unittest.TestCase`.

Reply:
ah, ok.